lib/rel.js

```diff
@@ -7,7 +7,8 @@ function relSitemapInject(data) {
 
   if (!rel || data.match(/rel=['|"]?sitemap['|"]?/i)) return;
 
-  const relSitemap = `<link rel="sitemap" type="application/xml" title="Sitemap" href="${url_for.call(this, path)}">`;
+  const relPath = typeof path === 'string' ? path : path[0];
```

Comment:
What if `path[0]` is txt and `path[1]` is xml?

Reply:
Thanks, I assumed the `index[0]` was more important before, but forgot it specified the `xml`.